themes/uob.typ

```diff
@@ -0,0 +1,131 @@
+//==============================================
+// University of Bristol theme for Typst slides.
+// Based on a previous version of David Barton's 
+// UoB LaTeX Beamer template, found at
+// https://github.com/dawbarton/UoB-beamer-theme
+// =============================================
+
+#let section = state("section", none)
+#let subslide = counter("subslide")
+#let logical-slide = counter("logical-slide")
+#let repetitions = counter("repetitions")
+#let global-theme = state("global-theme", none)
+
+#let new-section(name) = section.update(name)
```

Comment:
This need (and should) not be repeated here, user will write `#import "slides.typ": *` anyways.

Reply:
Oh sorry, I didn't mean importing it, I mean that the usage will be
```typ
#import "slides.typ": *
#import "themes/uob.typ": *

#show: slides.with(...)
...
```
So then all of those globals are already in scope.

Reply:
Turns out we don't get a circular dependency by importing slides.typ into uob.typ, so we can get the variables in scope within uob.typ with an import. Implemented that now.